test/e2e/multihoming.go

```diff
@@ -27,6 +27,14 @@ import (
 
 const PolicyForAnnotation = "k8s.v1.cni.cncf.io/policy-for"
 
+type Permission int
+
+const (
+	Specific Permission = iota
```

Comment:
btw maybe this approach is better than 
https://github.com/ovn-org/ovn-kubernetes/pull/4156/files#diff-8a0101c1714fa2f18fcb47cc2f71b4a820bc6e88fb23f0fcb557abafbfb3b924R825